sessions/js-fetch/star-wars/css/globals.css

```
@@ -0,0 +1,9 @@
+body {
```

```suggestion
:root {
  --nemo: #ff4a11;
  --granite: #252629;
  --water: #f3f5f9;
  --foam: #ffffff;
}

body {
```
We've started using these, and I think it'd be good to just stick to them - especially with limited color palettes.